requirements.txt

```diff
@@ -69,4 +69,5 @@ imap_tools==1.5.0  # Used by metagpt/tools/libs/email_login.py
 qianfan==0.3.2
 dashscope==1.14.1
 rank-bm25==0.2.2  # for tool recommendation
-jieba==0.42.1  # for tool recommendation
\ No newline at end of file
+jieba==0.42.1  # for tool recommendation
+flask   # for web api
```

Comment:
Make `flask` an extra dependency since it's not mandatory, See https://github.com/geekan/MetaGPT/blob/main/setup.py#L26.